test/test_jit_cuda_fuser.py

```diff
@@ -786,6 +786,18 @@ def t(x: torch.Tensor, y: torch.Tensor, z: torch.Tensor):
 
         z = torch.tensor([2], device="cuda").to(dtype_arg1)
 
+        if operation in [torch.pow]:
+            if dtype_arg1 == torch.int32 or dtype_arg2 == torch.int64:
```

Comment:
Does this test works for `torch.pow(int64_tensor, int32_tensor)`?

Reply:
Oops, good catch. This should have been disabled for when both inputs are integers. Let me update that.